File: Dia05/EliminarVocales.py
#Eliminar vocales en Python
def removeVowels(text):
    vowels = ["a", "e", "i", "o", "u", "á", "é", "í", "ó", "ú"]
    for vowel in vowels:
        while text.count(vowel) > 0:
            text = text.replace(vowel, "")
        while text.count(vowel.upper()):
            text = text.replace(vowel.upper(), "")
    return text

texto = input('Introduzca el texto para quitarle las vocales: ')
print(removeVowels(texto))

File: Dia05/EliminarVocales.js
//Eliminar vocales en JS
function removeVowels(text) {
    let vowels = ["a", "e", "i", "o", "u", "á", "é", "í", "ó", "ú"]
    vowels.forEach(element => {
        while (text.includes(element)) {
            text = text.replace(element,"")
        }
        while (text.includes(element.toUpperCase())){
            text = text.replace(element.toUpperCase(),"")
        }
    });
    return text
}

let texto = prompt('Introduzca el texto para quitarle las vocales')
alert(removeVowels(texto))